app/src/renderer/css/editor.css

```diff
@@ -119,6 +119,38 @@ video {
   }
 }
 
+
+.progress-bar-time-tip {
+  pointer-events: none;
+  position: absolute;
+  top: 0;
+  left: -50%;
+  margin-top: -24px;
+  transform: translate(-50%, 0);
+
+  background: rgba(0,0,0,.6);
```

Comment:
I'd add spaces between args.

Reply:
Also `.6` => `0.6` for readability reasons.

Reply:
I've generally gone for removing the `0` from decimal values starting with a 0 since it's more readable to me. But if you find it more readable, it's fine to go that way